packages/nx-plugin-openapi/src/executors/generate-api-lib-sources/executor.spec.ts

```diff
@@ -29,18 +32,28 @@ xdescribe('Command Runner Builder', () => {
         projects: {
           proj: {
             root: '',
-            sourceRoot: 'src',
+            sourceRoot: `./tmp/src/${dir}`,
             targets: {},
           },
         },
       },
       isVerbose: false,
-    };
+    });
   });
 
   it('can run', async () => {
-    const { success } = await executor(schema, context);
+    jest.setTimeout(30_000);
```

Comment:
Aren't this and the one on line 53 doing the same thing as the new 
(awesome) config prop in `packages/nx-plugin-openapi/jest.config.js`?

Reply:
i should remove this one, it actually just didn't work